styles/styles.css

```diff
@@ -0,0 +1,88 @@
+.flex-container {
+  display: flex;
+  flex-direction: column;
+}
+.flex-container > div {
+  margin: 10px;
+  padding: 20px;
+  font-style: normal;
+}
+h1 {
+  font-weight: bold;
+  color: red;
+}
+h2 {
+  font-weight: bold;
+  color: red;
+  font-style: italic;
+}
+.header {
+  margin: auto;
+}
+.navbar > ul li {
+  color: orange;
+  list-style: none;
+  text-align: center;
+  padding: 30px 15px;
+  display: block;
+}
+.navbar li {
+  float: left;
+}
+
+.navbar > li > a {
+  text-decoration: none;
+}
+.navbar a:hover {
+  color: magenta;
+  font-size: 150%;
+  font-family: monospace;
+}
+.navbar {
+  background-color: lightgrey;
+  width: auto;
+  padding: 30px;
+  margin: 10px;
+}
+.navbar1 ul li {
+  list-style: none;
+  text-align: center;
+  padding: 30px 15px;
+  display: inline;
+}
+.navbar1 > li > a {
+  text-decoration: none;
+}
+footer {
+  text-align: center;
+  color: white;
+  background-color: blue;
+}
+div.gallery {
+  margin: 5px;
+  border: 2px solid gray;
+  float: left;
+  width: 180px;
+}
+div.gallery img {
+  width: 100%;
```

Comment:
nice use of width for this. Also, when calling this the div isn't necessary simply 

`.galery img` will be enough